README.md

```diff
@@ -1,70 +1,79 @@
-# Getting Started with Create React App
+# Space-travelers-hub
+This is a React web application for a company that provides commercial and scientific space travel services. 
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![Screenshot](./screenshot1.png)
+![Screenshot](./screenshot2.png)
 
-## Available Scripts
 
-In the project directory, you can run:
+Additional description of the project and its features.
 
-### `npm start`
+## 🔴  Live Demo
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+[Space-travelers-hub](https://vigorous-jang-d9676c.netlify.app)
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Project Overview Video
 
-### `npm test`
+[Watch Here](https://www.loom.com/share/22b0607f2e7348cb9575c43d241d600c)
```

Comment:
- [ ] Talking about the video, I consider it important to make the video again, now follow these requirements when you work on this: 🐶 
- All the members should appear in the video and talk about the project.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
Hi  @vichuge 
May you please be a bit more specific with the following request?:

> * [ ]  At same time, it looks like you're not updating the local props once you press the button, please change the local props using useState. This change is for Missions.js too.

I am using this to get the state:
const missions = useSelector((state) => state.missionReducer);

and when Button is clicked an action is dispatched to update the state:
<Button variant="outline-danger" id={mission.mission_id} onClick={leave}>Leave&nbsp;Mission</Button>